packages/mobile/src/components/details-tile/DetailsTileAiAttribution.tsx

```diff
@@ -82,18 +59,30 @@ export const DetailsTileAiAttribution = ({ userId }: { userId: ID }) => {
 
   return user ? (
     <View style={styles.root}>
-      <View style={styles.title}>
-        <IconRobot fill={neutral} />
-        <Text style={styles.titleText}>{messages.title}</Text>
-      </View>
-      <Text style={styles.description}>
+      <Flex inline direction='row' alignItems='center' gap='s'>
+        <IconRobot color='default' />
+        <Text variant='label' style={{ marginTop: 2 }}>
```

Comment:
do you happen to know why the marginTop is necessary? what's causing the `alignItems='center'` to not work?

Reply:
the ai-atrribution icon is very bottom-heavy so it appears like the text is not aligned, so this is just a manual adjustment to make it seem better, kinda off, ill add a comment ^